pages/platform/ai/cli_18_howto_deploy_app/guide.en-gb.md

```diff
@@ -0,0 +1,283 @@
+      --auto-max-replicas <MAX REPLICAS>                                                                        Automatic scaling strategy: Maximum number of replicas [aliases: max-replicas]
+      --auto-resource-type <TYPE>                                                                               Automatic scaling strategy: Resource type to base the automatic scaling on [aliases: resource-type] [possible values: CPU, RAM]
+      --auto-resource-usage-threshold <THRESHOLD>                                                               Automatic scaling strategy: Average resource usage threshold (in percents) at which the number of replicas will be automatically increased [aliases: resource-threshold]
+      --probe-path <PATH>                                                                                       Path to access (defaults to /)
+      --probe-port <PORT>                                                                                       Number of the port to access (defaults to --default-http-port)
+      --no-color                                                                                                Remove colors from output
+  -h, --help                                                                                                    Print help (see more with '--help')
+```
+
+The `IMAGE` argument is mandatory. Indeed, you need to specify a Docker image that you either built yourself or find freely available on a public repository such as [DockerHub](https://hub.docker.com/). 
+
+**To launch a basic app, use the following command**:
+
+``` {.console}
+ovhai app run <registry-address>/<image-identifier>:<tag-name>
+```
+
+As indicated in the help command, many options can be passed to this command. Let's understand their usefulness and how to use them.
+
+### Setting environment variables
+
+You can tweak the behavior of your Docker image without having to rebuild it every time (like updating the input source of your model (camera, image, video, file, ...)) by using the `--env` flag. Using this, you can simply set environment variables directly in your app. 
+
+The values of these `--env` flags will take precedence over those specified in your Dockerfile and in your Python scripts, provided that the python variable is only initialized if it does not exist, as follows:
+
+``` {.console}
+if not os.environ["ENV_VAR"]:
+    os.environ['ENV_VAR'] = 'value'
+```
+
+As explained, this variable can be modified when the app is launched by using the `--env` flag:
+
+``` {.console}
+ovhai app run <registry-address>/<image-identifier>:<tag-name> \
+    --env ENV_VAR=new_val
+```
+
+### Assigning a name to the app 
+
+In order to manage your apps more easily, and to avoid ending up with random names, we recommend that you give your app a name. To do this, use the '--name' parameter in the following way:
+
+``` {.console}
+ovhai app run <registry-address>/<image-identifier>:<tag-name> \
+    --name segmentation_app
```

Comment:
I would have put a generic name to avoid confusion.
i.e.: `my_first_app` or something like that

```suggestion
    --name my_first_app
```